api/composer.json

```diff
@@ -74,5 +74,11 @@
     "incenteev-parameters": {
       "file": "config/parameters.yml"
     }
-  }
+  },
+  "repositories": [
+      {
+        "url": "https://github.com/acsauk/MinkExtension.git",
```

Comment:
Might be worth updating this to be in the MOJ account so that we have ownership if you move on at any point. Can be done as another ticket though.

Reply:
Yeah good point - will drop a ticket in now.

Reply:
https://opgtransform.atlassian.net/browse/DDPB-4184